lib/oli_web/components/delivery/instructor_dashboard.ex

```diff
@@ -74,21 +82,23 @@ defmodule OliWeb.Components.Delivery.InstructorDashboard do
                   hover:border-delivery-primary-200
                   focus:border-delivery-primary-200
                   #{if active, do: "!border-delivery-primary active", else: "border-transparent"}
-                "}>
-                <%= if is_function(label), do: label.(), else: label %>
-                  <%= if badge do %>
-                  <span class="text-xs inline-block py-1 px-2 ml-2 leading-none text-center whitespace-nowrap align-baseline font-bold bg-delivery-primary text-white rounded"><%= badge %></span>
-                  <% end %>
-              </.link>
-            </li>
-          <% end %>
-
-        </ul>
-      </div>
+                "}
+            >
+              <%= if is_function(label), do: label.(), else: label %>
+              <%= if badge do %>
+                <span class="text-xs inline-block py-1 px-2 ml-2 leading-none text-center whitespace-nowrap align-baseline font-bold bg-delivery-primary text-white rounded">
+                  <%= badge %>
+                </span>
+              <% end %>
```

Comment:
NIT: This looks like a good opportunity to replace that `<%= if badge do %>` for LiveView's new `:if` decorator 😁